GHzDACs/Cleanup/dac.py

```diff
@@ -677,8 +677,25 @@ def _setPolarity(self, chan, invert):
 
     @inlineCallbacks
     def _checkPHOF(self, op, fifoReadings, counterValue):
-        # Determine for which PHOFs (FIFO offsets) the FIFO counter equals
-        # counterValue.
+        """ Determine correct PHOF for FIFO counter of counterValue
+
+        If finds such a PHOF, sets PHOF and reads back FIFO counter to
+        verify that FIFO counter equals counterValue.
+
+        If no PHOF can be found to get an acceptable FIFO counter, PHOF will
+        be returned as -1, and success will be False.
```

Comment:
Generally try to keep things in the imperative. I suggest: "If no PHOF can be found to get an acceptable FIFO counter, return (-1, False)."